src/controls/OrbitControls.ts

```diff
@@ -89,6 +89,7 @@ class OrbitControls extends EventDispatcher {
     super()
 
     this.object = object
+    this.domElement = (null as unknown) as HTMLElement
```

Comment:
@joshuaellis this i don't know how to solve. domEl is not mandatory. but if it's defined as HTMLElement | undefined then that will mess up the entire file because each and every function now has to check it.

Reply:
it's not wrong though if you call `new OrbitControls` then call `OrbitControls.dispose` it'll break?